### ISSUE TITLE
lib frameworks not found in most recent download of Parse frameworks

### ISSUE THREAD

**Issue by erinjerri** [ParsePlatform/Docs#231]
![image](https://cloud.githubusercontent.com/assets/3184381/10456897/710645ea-7178-11e5-8184-491602ae44d2.png)

I cannot locate libz.dylib and libsqlite3.dylib when I download the frameworks and put these into XCode, which are needed to connect my app to Parse Platform. Help?

Are these the correct frameworks to include in lieu of the .dylib files? My app won't build (just get it hanging on the Simulator when I run a test build, I'm on El Capitan 10.11 + XCode 7 Beta 6:

![image](https://cloud.githubusercontent.com/assets/3184381/10456961/cb0f2b1a-7178-11e5-9ddf-72f4b0fbf742.png)

**Comment by ericnakagawa** [comment]
Hi Erin, you should be able to include libz.tbd and libsqlite3.tbd -- try compiling after doing this.

**Comment by nlutsenko** [comment]
Hey @erinjerri, you are right, we should update that for the latest Xcode.
The key principle here is the name of the library - `libz` or `libsqlite3` in our case.
These dynamic libraries are built into the platform and provided by Apple, we are simply linking with them.

.tbd files are new "text-based stub libraries", that provide a much more compact version of the stub libraries for use in the SDK, and help to significantly reduce its download size, as well as provide a better interface for using them in Swift.

.dylib are plain old dynamic libraries, which are still there behind the scenes of .tbd files.

Long story short, in order to resolve this, before we update the content on the quickstart, one of these 2 options:
- Use `libz.tbd` and `libsqlite.tbd` instead of dylibs
- Add `-lz` and `-lsqlite` to `Other Linker Flags` option in your target

**Comment by nlutsenko** [comment]
Fix is live in QuickStarts.

**Comment by erinjerri** [comment]
Thanks, ya I ended up reinstalling another way via Cocoapods (other devs
helped me out with it).

**Comment by nlutsenko** [comment]
That'll do it, it has the same libraries but links them automatically for you.
Glad you found the solution!
